src/components/descriptions/DescriptionsGenerator.ts

```diff
@@ -9,42 +9,48 @@
  * Generates the automatic descriptions.
  *
  * @export
- * @param {string} title
- * @param {object[]} data
- * @param {number} average
- * @param {string} context
- * @param {string} apiKey
- * @param {string} [model]
- * @param {number} [temperature]
- * @return {Promise<string[]>} An array with both longer and smaller descriptions.
+ * @return An array with both longer and smaller descriptions.
  */
-export async function generateDescriptions(
-	title: string,
-	data: object[],
-	average: number,
-	context: string,
-	apiKey: string,
-	model?: string,
-	temperature?: number,
-): Promise<string[]> {
-	data = JSON.stringify(data);
+export async function generateDescriptions({
+	title,
+	data,
+	average,
+	context,
+	apiKey,
+	model,
+	temperature,
+}: {
+	title: string;
+	data: object[];
+	average: number;
+	context: string;
+	apiKey: string;
+	model?: string;
+	temperature?: number;
+}): Promise<string[]> {
+	const dataString = JSON.stringify(data);
 	const key = apiKey;
```

Comment:
you can assign other names in the prop definition:
`apiKey as key`

would work just as well.